### DISCUSSION TITLE
DNAnalyzer's New Discord Server!!!

### DISCUSSION THREAD

**Discussion by VerisimilitudeX** [VerisimilitudeX/DNAnalyzer#256] [Announcements]
I started developing DNAnalyzer in July of 2022, and it quickly grew to have over 25 contributors, with more to come. I really appreciate everyone's support! It means miles to me every time a contributor asks to be assigned an issue and opens a PR. I am excited to see what is in store for DNAnalyzer in the next six months. Our team is pleased to announce the release of our new Discord server, where contributors and users will get the opportunity to collaborate on this project. I hope you will consider joining, and I look forward to continuing to work with everyone. Thank you.

DNAnalyzer Discord Server: https://discord.gg/X3YCvGf2Ug

**Comment by LimesKey** [comment]
I'm looking for a bot we can use to give contributors from Github a special role in the Discord, let me know if anyone knows one.

**Reply by ImpossibleReality** [reply]
We could probably make one, it wouldn't be that hard. It would be a pain to host though.

**Reply by LimesKey** [reply]
Can you not get a cheap VPS? Shouldn't need too much performance.

**Reply by VerisimilitudeX** [reply]
I can pay for hosting on Azure, if  you want to work on it @ImpossibleReality. Why would hosting be a pain though?
